python/tests/unit/arcticdb/test_string.py

```diff
@@ -161,3 +163,42 @@ def test_write_fixed_coerce_dynamic(lmdb_version_store):
     lmdb_version_store.write("strings", df, dynamic_strings=False)
     vit = lmdb_version_store.read("strings", force_string_to_object=True)
     assert_frame_equal(df, vit.data)
+
+
+def test_string_encoding_error_message(lmdb_version_store_tiny_segment):
```

Comment:
NIT: I'd make a test class and then put each commented block as a function so that pytest sees them as separate test cases.